tmt/steps/discover/fmf.py

```diff
@@ -147,6 +152,43 @@ def go(self):
         testdir = os.path.join(self.workdir, 'tests')
         dist_git_source = self.get('dist-git-source', False)
 
+        # Raise an exception if --fmf-id uses w/o url and git root
+        # doesn't exist for discovered plan
+        if self.opt('fmf_id'):
+            def print_error(plan_name=None):
```

Comment:
Name is a bit misleading - It's doesn't print error at all, only raises exception.
How about 'assert_git_url' ? (or something among those lines).